shared/reducers/chat.js

```diff
@@ -348,6 +348,18 @@ function reducer(state: Constants.State = initialState, action: Constants.Action
         )
       )
     }
+    case 'chat:setTypers': {
+      const {conversationIDKey, typing} = action.payload
+      // $FlowIssue
+      return state.update('conversationStates', conversationStates =>
+        updateConversation(
+          conversationStates,
+          conversationIDKey,
+          // $FlowIssue
+          conversation => conversation.update('typing', _ => Set(typing))
```

Comment:
sorry, i actually mean `conversation.set` instead of `conversation.update`. you're just writing the value so you don't need update since you're not referencing it